recipes/termcolor-cpp/bld.bat

```diff
@@ -0,0 +1,8 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ..
```

Comment:
might be useful to use `%CMAKE_ARGS%` as well, just in case

Reply:
only for windows build script? not needed on unix?

Reply:
you already have it on UNIX :)